_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -1,19 +1,20 @@
 .revenues_subpage-steps {
-  
+
   background-position: 0 0;
   background-size: 100%;
   padding-top: $base-padding;
-  
+
   &.onshore-oil{
     background: $onshore-oil-primary-color;
 
     .revenues_subpage-dyk_heading {
       background-color: $onshore-oil-secondary-color;
     }
-    .revenues_subpage-steps_heading {
+    .revenues_subpage-steps_heading,
```

Comment:
Selector `revenues_subpage-steps_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens